README.md

```diff
@@ -1,47 +1,75 @@
-# conu
-__CON__ -tainers __U__ -tils
-
-It provides low level __API__ for docker image/container testing.
-It is based on bash commands because primary target is to __test it
-as customers will__ use it.
-That's why it does not use python docker bindings.
-It provides python classes for:
-- __docker__ module
-  - __Image__ - image handling
-  - __Container__ - docker container handling
-- __utils__ module
-  - __Volume__ - it is mktemd utility on steroids
-  - __Probe__ - provides, simple port/file checks and waiting for this events
+# CONU
+
+Conu (CONtainer Utils) provides low level __API__ for image/container testing.
```

Comment:
my go:

    Conu is a library which makes it easy play with containers inside your code. It defines an API to access and manipulate containers, images and provides more, very helpful functions.

Reply:
my idea is to use __ConU __ - leads to __Con__tainer __U__tils abbreviation,  but it is up to you. I'm not against using simple __conu__ name

Reply:
@TomasTomecek  s/easy play/easy to play/
Anyway, since ConU was primarily meant for creating tests, that should also be the main message imho. So how about...
Conu/ConU/CONU is a library which makes it easy to write tests for your containers.
Or something like that.

Reply:
Honestly: once we release it, I'll start using conu everywhere, I'll probably paint my flat using conu.

+1 to @eliskasl's suggestion (even when I disagree with the message)